src/internal/analytics/__tests__/components/analytics-funnel.test.tsx

```diff
@@ -641,6 +641,48 @@ describe('AnalyticsFunnelSubStep', () => {
   });
 });
 
+describe('emit cloudscape-creation-flow-done event', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.useFakeTimers();
+    mockFunnelMetrics();
+  });
+
+  test('emit cloudscape-creation-flow-done event when funnelComplete', async () => {
+    let eventIsCalled = false;
+    document.addEventListener('cloudscape-creation-flow-done', () => {
```

Comment:
We should probably clean this up so it doesn't leak into other tests. Maybe move this up into the `describe` block (since the other test case uses it too), and then call `removeEventListener` when cleaning up